sites/friday/src/manage-resources/resource-init.md

```diff
@@ -106,79 +172,223 @@ platform integration:add \
 run the following command:
 
-```bash
+```bash {location="Terminal"}
 upsun integration:update --resources-init={{< variable "INITIALIZATION_STRATEGY" >}}
 ```
 
 For example, to use the `minimum` strategy for your deployment, run the the following command:
 
-```bash
+```bash {location="Terminal"}
 upsun integration:update --resources-init=minimum
 ```
 
 {{< /codetabs >}}
 
-### Minimum resources
+### Environment branch
 
-The following table shows the resources {{% vendor/name %}} allocates to your containers when you opt for the `minimum` [resource initialization strategy](#define-a-resource-initialization-strategy).
+By default, when you [branch an environment](/glossary.md#branch) to create a new child environment,
+the child environment inherits all the resources from its parent.
+However, you can set another [resource initialization strategy](#set-a-resource-initialization-strategy).
 
-| Container               | CPU  | RAM    | Disk*   |
-|-------------------------|------|--------|---------|
-| .NET                    | 0.1  | 64 MB  | 0 MB    |
-| Chrome Headless         | 0.1  | 64 MB  | None    |
-| Elasticsearch           | 0.1  | 448 MB | 256 MB  |
-| Elasticsearch Premium   | 0.1  | 448 MB | 256 MB  |
-| Elixir                  | 0.1  | 64 MB  | 0 MB    | 
-| Go                      | 0.1  | 64 MB  | 0 MB    |
-| InfluxDB                | 0.1  | 448 MB | 256 MB  |
-| Java                    | 0.1  | 448 MB | 0 MB    |
-| Kafka                   | 0.1  | 448 MB | 512 MB  |
-| Lisp                    | 0.1  | 64 MB  | 0 MB    |
-| MariaDB                 | 0.1  | 448 MB | 256 MB  |
-| Memcached               | 0.1  | 352 MB | None    |
-| MongoDB                 | 0.1  | 448 MB | 256 MB  | 
-| MongoDB Premium         | 0.1  | 448 MB | 256 MB  |
-| Network Storage         | 0.1  | 448 MB | 256 MB  | 
-| NodeJS                  | 0.1  | 64 MB  | 0 MB    |
-| OpenSearch              | 0.1  | 448 MB | 256 MB  | 
-| Oracle Java             | 0.1  | 448 MB | 0 MB    |
-| Oracle MySQL            | 0.1  | 448 MB | 256 MB  | 
-| PHP                     | 0.1  | 64 MB  | 0 MB    |
-| PostgreSQL              | 0.1  | 448 MB | 256 MB  | 
-| Python                  | 0.1  | 64 MB  | 0 MB    |
-| Rabbitmq                | 0.1  | 448 MB | 256 MB  | 
-| Redis ephemeral         | 0.1  | 352 MB | None    |
-| Redis persistent        | 0.1  | 352 MB | 256 MB  | 
-| Ruby                    | 0.1  | 64 MB  | 0 MB    |
-| Rust                    | 0.1  | 64 MB  | 0 MB    |
-| Solr                    | 0.1  | 448 MB | 256 MB  | 
-| Varnish                 | 0.1  | 448 MB | None    |
-| Vault KMS               | 0.1  | 448 MB | 256 MB  |
+{{% note theme="info" title="Prerequisite" %}}
 
-\* The disk size is set to `None` when the container never uses a disk, and to `0 MB` when the container doesn't require a disk but can use one.
+Make sure `sizing_api_enabled` is set to `true` in your `{{% vendor/configfile "app" %}}` file.
```

Comment:
We don't need to mention this prerequisite to the user.
It's more something internal that the user doesn't need to know.